sdk/servicebus/azure-servicebus/azure/servicebus/management/_xml_workaround_policy.py

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.core.pipeline import PipelineRequest
+from azure.core.pipeline.policies import SansIOHTTPPolicy
+
+
+class ServiceBusXMLWorkaroundPolicy(SansIOHTTPPolicy):
+    """A policy that mutates serialized XML to workaround ServiceBus requirement
```

Comment:
As before; I'd include here the tidbits you told us during your introduction (e.g. why this is needed, what it does by replacing the ns1 bits)

Reply:
Added an explaination.